Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -266,8 +266,9 @@ TextureCache::TCacheEntryBase* TextureCache::Load(const u32 stage)
 	const int texformat = tex.texImage0[id].format;
 	const u32 tlutaddr = tex.texTlut[id].tmem_offset << 9;
 	const u32 tlutfmt = tex.texTlut[id].tlut_format;
-	u32 tex_levels = (tex.texMode1[id].max_lod + 0xf) / 0x10 + 1;
-	const bool use_mipmaps = (tex.texMode0[id].min_filter & 3) != 0 && tex_levels > 0;
+	const bool mipmap_min_filter = (tex.texMode0[id].min_filter & 3) != 0;
+	u32 tex_levels = mipmap_min_filter ? ((tex.texMode1[id].max_lod + 0xf) / 0x10 + 1) : 1;
+	const bool use_mipmaps = mipmap_min_filter && tex_levels > 0;
```

Comment:
Yes, that's fine.  It's logically consistent, and it probably doesn't happen enough to make much of a difference.